massdriver-application-azure-app-service/dns.tf

```
@@ -1,6 +1,10 @@
+locals {
+  zone_name = element(split("/", var.dns.zone_id), length(var.dns.zone_id) - 2)
```

When I counted before, I believe 8 is the right index but def check my work.

```suggestion
  zone_name = element(split("/", var.dns.zone_id), 8)
```

---

I might store the split as a local and reference that in both places. We cant be sure that magic number will always work out.